thirdparty/detray/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ message(STATUS "Building Detray as part of the Acts project")
 set(DETRAY_VERSION "v${_acts_detray_version}")
 
 # Declare where to get Detray from.
-FetchContent_Declare(Detray ${ACTS_DETRAY_SOURCE})
+FetchContent_Declare(Detray ${ACTS_DETRAY_SOURCE} SYSTEM)
```

Comment:
I feat this requires `CMake >= 3.25` while we require `3.14` right now https://github.com/acts-project/acts/blob/17b7b92f58ce143a745fb2b671cf3d5785fa2676/CMakeLists.txt#L1

Reply:
Ah okay, I can add a case-statement for this.

Reply:
Okay I added the check, but sadly the failing CI runs version 3.22, not 3.25. :cry:

Reply:
I think there are ways to work around it by creating a new target and declaring it as `SYSTEM` manually. We had a similar discussion with GeoModel update a month ago which was then resolved by pushing it to pre-built dependencies. But I guess that is not an option here...